.markdownlint.yml

```
@@ -1,16 +1,22 @@
 # Default state for all rules
-default: true
+default: false
 MD003:
   # Heading style
   style: 'atx'
-MD004: false
-# Can't disable MD007 :/
-MD007: false
-MD009: false
-MD010:
-  code_blocks: false
-MD013: false
-MD024: false
-MD029: false
-MD033: false
-MD037: false # breaks on latex
+MD018: true
```

- Heading levels should only increment by one level at a time

I think this helps to ensure that headings make sense / are consistent

```suggestion
MD001: true
MD018: true
```